articles/quickstart/webapp/rails/01-login.md

```diff
@@ -84,6 +98,10 @@ end
 
 We need a way for users to trigger authentication. Add a link to `/auth/auth0` anywhere in an existing template or use the steps below to generate a homepage in a new app.
 
+::: note
```

Comment:
Can we make this a warning instead? (yellow background, to grab more the attention)

```suggestion
::: warning
```